Redraw atrous area fixes

jenshannoschwalm opened the pull request:
The area should also be redrawn if
- the UI pipe has finished while atrous is in focus
- when the module gets focus

Fixes #19886

TurboGit commented on src/iop/atrous.c:
```diff
@@ -1766,6 +1766,23 @@ const dt_action_def_t _action_def_equalizer
+  dt_iop_atrous_gui_data_t *g = self->gui_data;
+  if(!g || darktable.gui->reset) return;
+
+  if(dt_dev_gui_module() == self)
```
Maybe we want to refresh if the module is expanded and enabled? Because if you open a another IOP and still see the contrast equalizer UI it seems better to also refresh it?

jenshannoschwalm committed:
Redraw contrast equalizer area fixes

The area should also be redrawn when
- the UI pipe has finished while atrous is enabled and expanded
- the contrast equalizer gets focus

TurboGit reviewed (approved):
Looks good to me, thanks!
